pom.xml

```diff
@@ -143,8 +143,8 @@ flexible messaging model and an intuitive client API.</description>
     <postgresql-jdbc.version>42.2.12</postgresql-jdbc.version>
     <clickhouse-jdbc.version>0.2.4</clickhouse-jdbc.version>
     <mariadb-jdbc.version>2.6.0</mariadb-jdbc.version>
-    <hdfs-offload-version3>3.2.0</hdfs-offload-version3>
-    <org.eclipse.jetty-hdfs-offload>9.3.24.v20180605</org.eclipse.jetty-hdfs-offload>
+    <hdfs-offload-version3>3.3.0</hdfs-offload-version3>
+    <org.eclipse.jetty-hdfs-offload>${jetty.version}</org.eclipse.jetty-hdfs-offload>
```

Comment:
Could we just remove the special variable and use `${jetty.version}` directly?

Reply:
okay.
updated